src/main/lib/file-utils.js

```diff
@@ -17,28 +20,54 @@ const readJson = (filename, processFn) => {
 const writeJson = (filename, json) => {
   return new Promise((resolve) => {
     fs.writeFile(filename, json, (err) => {
-      if (err) throw err;
+      if (err) {
+        throw err;
+      }
       resolve();
     });
   });
 };
 
-const exists = (path) => {
-  return fs.existsSync(path);
-};
+const exists = (path) => fs.existsSync(path);
 
-const listFilesInDirectory = (dir, excludes = []) => {
-  return fs.readdirSync(dir, { withFileTypes: true })
-    .filter((file) => {
-      if (file.isDirectory()) {
-        return true;
-      } else {
-        return path.extname(file.name) === '.json';
-      }
-    })
-    .filter((file) =>
-      !excludes.some(el => matcher.isMatch(file.name, el))
-    );
+const getJsonFiles = (directory, exclusions = []) => {
+  let paths = glob.sync(directory + '/**/*.json');
+  let relativePaths = toRelativePaths(paths, directory);
+  exclusions = exclusions.map(exclusion => exclusionUtils.prepareExclusion(exclusion));
+  let filteredPaths = exclusionUtils.filterPaths(relativePaths, exclusions);
+  return groupToSheets(filteredPaths);
 };
 
-module.exports = { writeJson, readJson, listFilesInDirectory, exists };
+function toRelativePaths (array, root) {
+  return array.map(
+    file => path.relative(root, file));
+}
+
+function groupToSheets (paths) {
+  return paths.reduce((groupMap, filePath) => {
+    // Split on the first '/'
+    let splitPath = filePath.split(/\/(.+)/, 2);
+
+    if (splitPath.length > 1) {
+      prepareMap(groupMap, splitPath[0]);
+      groupMap[splitPath[0]].push(splitPath[1]);
+    } else {
+      prepareMap(groupMap, splitPath[0]);
+    }
+
+    return groupMap;
+  }, {});
+}
+
+function prepareMap (map, key) {
```

Comment:
Bad name and useless IMO. Could be just inlined into `groupToSheets` .e.g.

```
groupMap[splitPath[0]] = []
if (splitPath.length > 1) {
  groupMap[splitPath[0]].push(splitPath[1]);
}
```

I'd also name `splitPath[0]` and `splitPath[1]` so that it is easier to understand what is going on.